scripts/postprocess.py

```diff
@@ -240,14 +239,20 @@ def _parse_line_based(fout, bench_name, nodes):
         # fail.
         alg_name = bench_name[0 : bench_name.find("_dlaf")]
 
-        if alg_name in ["chol", "hegst", "trsm", "trmm"]:
+        print(alg_name)
+
+        if alg_name in ["chol", "hegst", "trmm"]:
             pstr_res = "[{run_index:d}] {time:g}s {perf:g}GFlop/s{matrix_type:optional_text} ({matrix_rows:d}, {matrix_cols:d}) ({block_rows:d}, {block_cols:d}) ({grid_rows:d}, {grid_cols:d}) {:d}{backend:optional_text}"
-        if alg_name in ["red2band", "band2trid", "bt_band2trid", "bt_red2band"]:
+        if alg_name in ["trsm"]:
+            pstr_res = "[{run_index:d}] {time:g}s {perf:g}GFlop/s{matrix_type:optional_text} ({matrix_rows:d}, {matrix_cols:d}) ({first_eval_idx:d}, {last_eval_idx:d}) ({block_rows:d}, {block_cols:d}) ({grid_rows:d}, {grid_cols:d}) {:d}{backend:optional_text}"
```

Comment:
This is not a blocking comment, but I want to point out that the new entries here will prevent parsing of old benchmark results. Ok to simply require using an old commit of the parsing scripts for that?

Reply:
This is a very good point. I can add an option to trigger the old behavior if it makes things easier going forward.

Reply:
Still not requesting this change for this PR, but if you want to look into it, you might want to have a look at the `optional_parser` in `postprocess.py`. We added that the previous time we changed the output format. It will parse the field to `None` if it's not there (though I see we don't even use the potentially-parsed field at the moment...).

Reply:
I actually need this to compare with the `master` branch, therefore I implemented it as you suggest. Thanks for pointing it out.